src/features/calendar/components/utils.ts

```diff
@@ -3,12 +3,30 @@ import dayjs from 'dayjs';
 
 import { NonEventActivity } from 'features/campaigns/hooks/useClusteredActivities';
 import range from 'utils/range';
+import { removeOffset } from 'utils/dateUtils';
 import {
   ACTIVITIES,
   CampaignActivity,
   EventActivity,
 } from 'features/campaigns/types';
 
+export function isAllDay(start: string, end: string): boolean {
+  const startDate = new Date(removeOffset(start));
+  const endDate = new Date(removeOffset(end));
+
+  // Check if the start and end dates are not on the same day
+  if (startDate.toDateString() !== endDate.toDateString()) {
+    // If start time and end time are 00:00:00 return true
+    if (
+      startDate.toString().split(' ')[4] == '00:00:00' &&
```

Comment:
This is a little unrelated as your PR only moves this function from one file to another, but I'm curious why this function works and maybe it's an opportunity for me to learn about some of the horrible grubby little details of how Zetkin handles dates.

An all-day event in Sweden for e.g. today would have a UTC start date of  2024-03-15 23:00:00 and an end date of 2024-03-16 23:00:00. So the `startDate.toString().split(' ')[4] == '00:00:00'` check would return `false `and as a result the event would be mis-classified as not an all-day event.

Struggling to figure out independently why this isn't an issue with Zetkin.